examples/plotting_examples--airmass.ipynb

```diff
@@ -0,0 +1,1240 @@
+       "1BZnKSLGuKbUHCQtL+lySbdJulXSXn2kWUTSWZJukjRZ0oQe22eXdKOkc5qR53YgsWC9z8K5wE+A\n",
+       "tyUwREQzNKtZ6UVgH9sTgA2BPSWt0SPNAcANttcGdgaO6LF9b8p9B8b8uP06Z+FDlM87HzDB5oTM\n",
+       "WYiIZmlKcLD9sO0p9fF0YCqwTI9kawCX1zR/A8ZLWgJA0nKUGb/HAGpGnltFYjylpvAtYAebT9pM\n",
+       "a22uIqLTNL1DWtJ4ykibyT023QRsV9NsALweWK5uOwz4IozdO5TVRfK+BFwHXAOsY3NVi7MVER2q\n",
+       "qcFB0vzA6cDetQbR6GBgYUk3Ap+lrCT6iqRtgEdt38gYrTVIbARcD2xBmcj2XZsXWpytiOhgspvT\n",
+       "jC1pHKW55ALbhw8i/T2U+w/sD3wceIlyn+MFgTNs79wjvYFvNrw0yfak4cn9yJBYBPge8F7KInlZ\n",
+       "CykiRoykicDEhpe+YbvPi+6mBAdJAk4Aptnep580CwHP2X5B0u7AJrZ36ZFmc2A/29v2sb/7+5Dt\n",
+       "RkLA9sAPgd8D+9v8u7W5iohOM9B5s1nzHDYBPgbcXJuNoIxOWgHA9tHAmsDxtQZwK/DJft5rVF9Z\n",
+       "S6wM/BR4HfDB3L85ItpR05qVRlq71xwk5gT2rT+HAIfbvNjaXEVEJ2uHmkNHk9gEOBr4J7C+zb2t\n",
+       "zVFExMASHEZQ7XA+GNgG2Ac4LR3OETEaZOG9EVBnOO9AuSvbS5S7suV2nRExaqTmMMwaOpyXBraz\n",
+       "+UuLsxQRMWSpOQwTiTkl9qfM/L6U0reQwBARo1JqDsMgHc4RMdYkOMyChhnO2wKfB05Pv0JEjAVp\n",
+       "VpoJPTqcX6Z0OGckUkSMGak5DJHESpQO52VIh3NEjFGpOQxSXVL7K8BfKfed+J8EhogYq1JzGIS6\n",
+       "pPYvgAeAN9vc0+IsRUSMqASHAUgsDHwXeD9lhnMmskVER0izUh9qh/NHKB3Os1Pu4Zx7LUREx0jN\n",
+       "oYd6D+ejKLcp/YjNNa3NUURE86XmUNUO5y9S7uF8NbBeAkNEdKrUHACJt1BmOD8KbGjzjxZnKSKi\n",
+       "pTo6OEgsSOlw/iDlJjynpF8hIqJDm5Vqh/N2wO3AXJQO55MTGCIiio6rOUisABwJrALsYHNVi7MU\n",
+       "EdF2OqbmIDGHxBeAGyiznNdJYIiI6FtH1Bwk3kzpcH4C2MjmzhZnKSKirY3pmoPEghI/Bs4BDgPe\n",
+       "kcAQETFjYzI4NHQ43wbMS+lwPjEdzhERgzPmmpVqh/NPgFWBnWyubHGWIiJGnTFVc2jocL6W0uGc\n",
+       "wBARMROaEhwkLS/pckm3SbpV0l59pFlE0lmSbpI0WdKEhm0LSzpd0lRJt0vasJ9DbUXpcD7I5vmR\n",
+       "+jztTtLEVuehXaQsuqUsipTD4DSr5vAisI/tCcCGwJ6S1uiR5gDgBttrAzsDRzRsOwI43/YawFrA\n",
+       "1H6Okw7nYmKrM9BGJrY6A21kYqsz0CYmtjoDo0FTgoPth21PqY+nU07uy/RItgblDmvY/hswXtIS\n",
+       "khYCNrP9q7rtJdtP9X2cdDhHRAyHpvc5SBoPrAtM7rHpJmC7mmYDypLZywErAo9JOk7SDZJ+KWne\n",
+       "5uU4IqLzyG7exbak+YFJwEG2z+6xbQFK89G6wC3A6sBuwJzAn4GNbV8r6XDgadtf77F/ag0REUNk\n",
+       "W3293rTgIGkccC5wge3DB5H+HuBNwPzAn22vWF/fFPiK7W1GMr8REZ2sWaOVBBwL3N5fYJC0kKQ5\n",
+       "6+PdgStsT7f9MHC/pFVr0i0pk9siImKENKXmUK/2rwRuhlc7jQ8AVgCwfbSkjYDj6/ZbgU92dTxL\n",
+       "Whs4htLEdBewa3+d0hERMeua2ucwHCTNZvuVVuejHaQsuqUsuqUsuqUsug21LEbFDGlJ80v6nKSV\n",
+       "gbnra312oox1KYtuKYtuKYtuKYtus1IWbV9zkPR24GeUpqbHgRdsf661uWqNlEW3lEW3lEW3lEW3\n",
+       "WS2L0VBzWAY4xfYHga8Bm0j6JJRqUktz1nwpi24pi24pi24pi26zVBZtV1iSVpC0XsNLqwPPAth+\n",
+       "FPgy8O36fEy3JaYsuqUsuqUsuqUsug17Wdhumx/gIOB+4BLgUGBhYBPg7h7p/gB8rdX5TVmkLFIW\n",
+       "KYt2+BmJsmibmoOkxSn3YFgF+AjwEvAN29cAUyV9tyH5r4Cl6sS6MSdl0S1l0S1l0S1l0W2kyqJt\n",
+       "ggNl5dYNgSVsPwmcCiDp48AewE6S3lrTrgY8aPvFluR05KUsuqUsuqUsuqUsuo1IWbQsOHQNp5I0\n",
+       "uyS5TGo7FdipJrmFsqbSRsAjwDeBHSVdWdNc2/xcj4yURbeURbeURbeURbemlUUL2sY+BawDLNjH\n",
+       "tvdRltl4U32+PnA2sHB9Pg+wdavb91IWKYuURcpirJdF02oOkiZImgJsUz/kjxu2nSjpzcB1lOUx\n",
+       "Pg9g+zpgMWDp+vw52+c1K88jJWXRLWXRLWXRLWXRrWVl0cSo9zbgZ/Xx/JQVWg+
```

Comment:
I think you either want this to be ``plt.legend`` instead of ``ax.legend`` (`plt.legend()` will use the current axis, which in this case is the one you want), or you want to set the `ax` variable to what's returned by any of the `plot_airmass` call.